docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,663 @@
+# Performing a Cross-Chain Swap with PKPs and Lit Actions
+In this section, we will demonstrate how you can use PKPs and Lit Actions to perform a cross-chain swap. There are three parts to this flow:
+
+- Using Lit PKPs as escrow
+- Using Lit Actions to define swap conditions
+- Implementing Mint/Grant/Burn with SDK
+
+## 1) Using Lit PKPs as escrow
+
+[PKPs](../user-wallets/pkps/overview) are decentralized key pairs generated by the Lit Network. Using Lit Actions, we can program them to sign transactions under specific conditions, enabling escrow mechanisms. We’ll explore how to execute a cross-chain swap with this setup.
+
+### Example:
+Suppose Alice holds 100 token A on Base and Bob holds 100 token B on Chronicle and they both agree to a swap of,
+- 4 token A from Alice to Bob on Base
+- 8 token B from Bob to Alice on Chronicle
+
+We have an [example code](https://github.com/anshss/lit-evm-swap) for the below solution.
```

Comment:
> We have an [example code](https://github.com/anshss/lit-evm-swap) for the below solution.

"A complete code example can be found [here](https://github.com/anshss/lit-evm-swap)"

- However, let's first ensure that we move the referenced example to the Lit Protocol repo first (in developer guides?)

Reply:
yup, moved it